staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -97,6 +98,8 @@ func (f *FieldManager) Update(liveObj, newObj runtime.Object, manager string) (r
 		return nil, fmt.Errorf("failed to create typed live object: %v", err)
 	}
 	apiVersion := fieldpath.APIVersion(f.groupVersion.String())
+	manager = f.buildManagerInfo(manager, f.groupVersion.String())
```

Comment:
since `buildManager` is a method of the `FieldManager`, maybe you don't have to pass the `f.groupVersion`?

Reply:
right, I wanted to ask if the f.groupVersion is even the right one (of the new object).